opentech/apply/workflow.py

```diff
@@ -5,8 +5,18 @@
 from django.forms import Form
 from django.utils.text import slugify
 
+"""
+This file defines classes which allow you to compose workflows based on the following strucutre:
+
+Workflow -> Stage -> Phase -> Action
+"""
+
 
 class Workflow:
+    """
+    Holds the stages of a workflow. When a stage finishes will return the next stage in the workflow
```

Comment:
nitpicking: 
> A Workflow is a collection of Stages an application goes through. When a Stage is complete, it will return the next Stage in the list or `None` if no such Stage exists.